tica_non_rev

fnueske opened the pull request:
- Activated non-reversible option in TICA. Disabled kinetic_map, commute_map and weights=="koopman" for non-reversible TICA
- Adjusted TICA tests
- Fixed documentation errors in msm.estimate_markov_model

Commit:
Activated non-reversible TICA. Fixed documentation erros for estimate_msm.

Commit:
Merge branch 'devel' into tica_non_rev

franknoe reviewed (commented):
TICA looks good. 

What are the doc changes in pyemma/msm/api.py for?

fnueske commented:
I just corrected a mismatch between function signature and docstring. One parameter was called "dt_traj" in the signature and "dt" in the docstring. I also noticed that "estimate" was no longer a parameter in the signature, so I removed it from the docstring.

franknoe commented:
ok, thanks. Then this is good for merging.

Am 08/05/17 um 18:42 schrieb Feliks Nüske:
>
> I just corrected a mismatch between function signature and docstring. 
> One parameter was called "dt_traj" in the signature and "dt" in the 
> docstring. I also noticed that "estimate" was no longer a parameter in 
> the signature, so I removed it from the docstring.
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub 
> <https://github.com/markovmodel/PyEMMA/pull/1078#issuecomment-299921470>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AGMeQk1tNC6q3K73ZCmWztVqhFfS81Xiks5r30X-gaJpZM4NTY4p>.
>


-- 

----------------------------------------------
Prof. Dr. Frank Noe
Head of Computational Molecular Biology group
Freie Universitaet Berlin

Phone: (+49) (0)30 838 75354
Web:   research.franknoe.de
Mail:  Arnimallee 6, 14195 Berlin, Germany
----------------------------------------------

marscher commented:
Thank you!
